docs/site/Database-migrations.md

```diff
@@ -20,8 +20,34 @@ LoopBack offers two ways to do this:
 - **Auto-update**: Change database schema objects if there is a difference
   between the objects and model definitions. Existing data will be kept.
 
-For connector-specific settings for defining model schemas and auto-migration,
-check out the specific connector under
+Part of the database schema definition can be specified via the model and / or
+property definition. For example, the following settings is a common definition
+that indicates the data type and the length of a column and also uses a column
```

Comment:
This is defined in the `postgresql` part instead of the `settings` portion, or do you consider them both `settings`?

Reply:
I will remove
```
@model({
  settings: {
    postgresql: {schema: 'public', table: 'mymodel'},
  },
})
```
 part to make it clear.